src/main/java/uk/gov/crowncommercial/dts/scale/cat/utils/CSVFileUtils.java

```diff
@@ -0,0 +1,62 @@
+package uk.gov.crowncommercial.dts.scale.cat.utils;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.apache.commons.csv.CSVFormat;
+import org.apache.commons.csv.CSVParser;
+import org.apache.commons.csv.CSVRecord;
+import org.springframework.web.multipart.MultipartFile;
+
+import uk.gov.crowncommercial.dts.scale.cat.model.jaggaer.RfxTemplateMapping;
+
+public class CSVFileUtils {
+	  public static String TYPE = "text/csv";
```

Comment:
Should be `public static final` to make really constant/immutable.
TYPE is very generic maybe `CONTENT_TYPE` would be better?